### PR TITLE
Disabled Zafira should not fail build

### PR DESCRIPTION
**Pull request by ilya-varchenya**

### CODE REVIEW

File: src/com/qaprosoft/jenkins/Utils.groovy [hunk 1]
```diff
@@ -64,4 +64,32 @@ class Utils {
     static boolean getBooleanParameterValue(parameter, currentSuite){
         return !isParamEmpty(currentSuite.getParameter(parameter)) && currentSuite.getParameter(parameter).toBoolean()
     }
+
+    static def getZafiraServiceUrl(orgName) {
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/Utils.groovy]: I see absolutely the same methods. How about moving common code into the 3rd one?

File: src/com/qaprosoft/jenkins/pipeline/Organization.groovy [hunk 2]
```diff
@@ -261,8 +265,10 @@ class Organization {
 		//TODO: remove mcloud registration in released version
 		logger.debug("mcloud: " + "http://demo:demo@\${QPS_HOST}/mcloud/wd/hub")
 		registerHubCredentials(folder, "mcloud", "http://demo:demo@\${QPS_HOST}/mcloud/wd/hub")
-		
-		registerZafiraCredentials(folder, Configuration.get(Configuration.Parameter.ZAFIRA_SERVICE_URL), Configuration.get(Configuration.Parameter.ZAFIRA_ACCESS_TOKEN))
+
+        if (!isParamEmpty(zafiraServiceURL)) {
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/pipeline/Organization.groovy]: let's add verification onto the empty for zafiraAccessToken as well here!

File: src/com/qaprosoft/jenkins/pipeline/Repository.groovy [hunk 3]
```diff
@@ -106,8 +106,8 @@ class Repository {
                 def zafiraFields = Configuration.get("zafiraFields")
                 logger.debug("zafiraFields: " + zafiraFields)
                 if (!isParamEmpty(zafiraFields) && zafiraFields.contains("zafira_service_url") && zafiraFields.contains("zafira_access_token")) {
-                    def zafiraServiceURL = Configuration.get(Configuration.Parameter.ZAFIRA_SERVICE_URL)
-                    def zafiraRefreshToken = Configuration.get(Configuration.Parameter.ZAFIRA_ACCESS_TOKEN)
+                    def zafiraServiceURL = getZafiraServiceUrl('qaprosoft')
```
Comment (vdelendik) [inline src/com/qaprosoft/jenkins/pipeline/Repository.groovy]: why organization is hardcoded?

File: src/com/qaprosoft/jenkins/Utils.groovy (hunk 1 above)
Comment (ilya-varchenya) [reply src/com/qaprosoft/jenkins/Utils.groovy]: I created new getZafiraServiceParameters instead of those two

**Comment by vdelendik** [comment]
too dangerous

**Pull request closed without merging**